monitoring/managing-metrics.adoc

```diff
@@ -34,19 +34,27 @@ include::modules/monitoring-specifying-how-a-service-is-monitored.adoc[leveloffs
 include::modules/monitoring-querying-metrics.adoc[leveloffset=+1]
 //include::modules/monitoring-contents-of-the-metrics-ui.adoc[leveloffset=+2]
 include::modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation] for more information about creating PromQL queries.
+
 include::modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
 
-* See the xref:../monitoring/managing-metrics.adoc#querying-metrics-for-user-defined-projects-as-a-developer_managing-metrics[Querying metrics for user-defined projects as a developer] for details on accessing non-cluster metrics as a developer or a privileged user
+* See the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation] for more information about creating PromQL queries.
 
 include::modules/monitoring-exploring-the-visualized-metrics.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
 
-* See the xref:../monitoring/managing-metrics.adoc#querying-metrics_managing-metrics[Querying metrics] section on using the PromQL interface
+* See xref:../monitoring/managing-metrics.adoc#querying-metrics_managing-metrics[Querying metrics] for details on using the PromQL interface
```

Comment:
Same comments as above, I think it's easier to scan and click if the links are at the end of the sentence. 

However, the change is not needed :)

Reply:
This was existing content that I moved, but I agree it reads better with your edits. I'll make the changes